feature: add parallel decorator for functions preprocess, score, and filter (#49)
Helsinki-NLP/OpusFilter: README.md

The syntax for the `opusfilter` is
```

> opusfilter [--overwrite] [--last LAST] [--single SINGLE] [--n_jobs N_JOBS] CONFIG

Reviewer: ```suggestion
opusfilter [--overwrite] [--last LAST] [--single SINGLE] [--n-jobs N_JOBS] CONFIG
```

(No underscores in options for consistency.)

only a single step (`--single`). It the latter, the user has to
make sure that all input files for the step already exist. The first
step has number 1, and -1 points to the last step, -2 to the second to

> last, and so on. The `--n_jobs` option indicate number of processes to use when running `score`, `filter` and `preprocess` steps. This value will overwrite `default_n_jobs` in `common` section below.

Reviewer: ```suggestion
last, and so on. The `--n-jobs` option indicate number of processes to
use when running `score`, `filter` and `preprocess` steps. This value will 
overwrite `default_n_jobs` in the `common` section.
```

  (with `filterfalse` option) and `score` steps. Increasing the value
  from the default 100000 may speed up things at the cost of increased
  memory use.

> * `default_n_jobs` for parallel run `score`, `filter` and `preprocess` steps. It indicate the number of processes to uese. The default value is 1.

Reviewer: ```suggestion
* `default_n_jobs` for defining the number of parallel processes to use
  for `score`, `filter`, and `preprocess` steps. The default value is 1.
```
